index.html

```diff
@@ -40,6 +41,161 @@ <h1>I'm Hossain.<br>Glad to see you! </h1>
         <li><a href="#" title="Angellist"><img src="img/angellist.svg" alt="angellist logo"></a></li>
       </ul>
     </section>
+     <!-- Work -->
+    <section id="work">
+      <div class="work-first flex-column">
+        <div class="work-description flex-column">
+          <div> <img src="img/Portfolio-1.png" alt="potfolio image"></div>
+          <div>
+            <div class="specification">
+              <h2>Hossain</h2>
+              <div class="flex">
+                <h4>CANOPY</h4>
+                <ul class="flex">
+                  <li>Full Stack Dev</li>
+                  <li>2023</li>
+                </ul>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="work-description">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="work-description flex">
+          <ul class="flex">
+            <li class="language-tag flex"><p>HTML</p></li>
+            <li class="language-tag flex"><p>CSS</p></li>
+            <li class="language-tag flex"><p>Javascript</p></li>
+          </ul>
+        </div>
+        <button class="see-project-btn">See Project</button>
```

Comment:
👍 Great job on the implementation! However, there are some improvements needed for the interactive properties of your buttons to align with the design. Here are the suggestions:

- [ ] Hover effect: In the design, the hover color for the button is specified as `#6070FF`, but in your code, you used `#24245c`. Please update the hover color to match the design. Additionally, the design includes a box shadow effect on the buttons, which is currently missing in your implementation. Consider adding the appropriate box shadow styles to match the design.

- [ ] Active effect: According to the design, the active stage color should be `#2230D2`, but you currently have `#0505b3`. Please update the active color to match the design specifications.

By following the design guidelines and incorporating the correct colors and box-shadow styles, you'll ensure that your buttons are visually consistent and aligned with the intended design. Keep up the great work! 👍🎨